Sources/UIKit/NSLayoutConstraint.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  NSConstraintLayout.swift
+//  Alicerce
+//
+//  Created by Tiago Veloso on 15/02/2018.
+//  Copyright © 2018 Mindera. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension NSLayoutConstraint {
+
+
+    /// Similar to NSLayoutConstraint.activate(), but this method ensures that the views
+    /// that are adding the constraints disable translatesAutoresizingMaskIntoConstraints
+    ///
+    /// - Parameter constraints: an array of constrains to activate
+    public static func add(constraints: [NSLayoutConstraint]) {
+
+        constraints.forEach {
+            if let v = $0.firstItem as? UIView {
+
```

Comment:
please be consistent with vertical whitespaces 💄

Reply:
💄  can we call it `let view` instead of `let v` ?